oneconfigquarantine/build.gradle.kts

```diff
@@ -42,7 +42,7 @@ dependencies {
 		mappings("de.oceanlabs.mcp:mcp_stable:22-1.8.9")
 		forge("net.minecraftforge:forge:1.8.9-11.15.1.2318-1.8.9")
 
-		modApi("cc.polyfrost:oneconfig-1.8.9-forge:0.1.0-alpha+") // Don't you just love 0.1.0-alpha+
+		modApi("cc.polyfrost:oneconfig-1.8.9-forge:0.2.0-alpha+") // no
```

Comment:
Yeah please stop making that the only way of including your mod and allow us to use proper versions. You are lowkey blocking us from using dependency verification to verify we arent pulling in maliciously modified updates. Just have a `1.0.0` version and increment that like a normal person. This is disgraceful for a project that *aims to be a library*.

Reply:
too much trolling